### DISCUSSION TITLE
Stream Deck - Flight Tracker - create a GPS enable / disable button

### DISCUSSION THREAD

**Discussion by Basilrug** [nguyenquyhy/Flight-Tracker-StreamDeck#424] [Q&A]
Hello, can anyone help me to create  the above button for GA aircraft in MSFS, like the Cessna, TBM 930 Cirrus Jet etc?
I want to add a button to my Stream Deck Plus profile, that turns GPS on or off.  I can't see to figure out the right way to do this.  Thanks for any help
Capt Coop

**Comment by ThermoLarry** [comment]
Hello, I have the same question.  I believe we need to use the LVARs something like this:

L:XMLVAR_GPS_DISABLED_AS530_1
L:XMLVAR_GPS_DISABLED_AS430_1

But I cannot get them to work so far.  I also have the Mobiflight WASM installed for information.

**Comment by Basilrug** [comment]
Hi ThermoLarry  I did find that one of the free G100 autopilot profiles at flightsim.to works perfectly and has a NAV button that works.. the variable in there is AP_HDG_HOLD   Not sure if that's what you are looking for?  Cheers.
